Update Storage Service Encryption (SSE) with option to encrypt storag…

laraaleite opened the pull request:
…e account with customer managed keys

Update Storage Service Encryption (SSE) with option to encrypt storage account with customer managed keys
https://azure.microsoft.com/en-gb/blog/announcing-storage-service-encryption-with-customer-managed-keys-ga/
https://docs.microsoft.com/en-us/azure/storage/common/storage-service-encryption-customer-managed-keys

By default, data is encrypted using Microsoft Managed Keys for Azure Blobs, Tables, Files and Queues. You may choose to bring your own key for encryption for Azure Blobs and Files. Encryption for Tables and Queues will always use Microsoft Managed Keys.
Please note that after enabling Storage Service Encryption, only new data will be encrypted, and any existing files in this storage account will retroactively get encrypted by a background encryption process.

azurecla commented:
Hi __@troyanov__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
@jessebenson - Could you please review this contribution from @troyanov and let me know if we can merge it?

jessebenson commented:
:+1: Looks good!

tysonn commented:
@troyanov - Thanks for the update!

laraaleite committed:
Update Storage Service Encryption (SSE) with option to encrypt storage account with customer managed keys

Update Storage Service Encryption (SSE) with option to encrypt storage account with customer managed keys
https://azure.microsoft.com/en-gb/blog/announcing-storage-service-encryption-with-customer-managed-keys-ga/
https://docs.microsoft.com/en-us/azure/storage/common/storage-service-encryption-customer-managed-keys

By default, data is encrypted using Microsoft Managed Keys for Azure Blobs, Tables, Files and Queues. You may choose to bring your own key for encryption for Azure Blobs and Files. Encryption for Tables and Queues will always use Microsoft Managed Keys.
Please note that after enabling Storage Service Encryption, only new data will be encrypted, and any existing files in this storage account will retroactively get encrypted by a background encryption process.
